### ISSUE TITLE
Midi player on Safari

### ISSUE THREAD

**Issue by mirbon** [SonyCSL-Paris/LeadsheetJS#10]
The 'Midi player' example has problems on Safari (iOS and OS X): the notes and the music are not synchronized and the 'time' are not constant.

**Comment by vinzentt** [comment]
Right. We mainly use Chrome but I will try to check why it is not synchronized on Safari.

**Comment by mirbon** [comment]
Hi, are there any news about the problem with Safari?
Many thanks.

**Comment by vinzentt** [comment]
Hi, 
nope not yet. We need to change completely the way we generate midi. It is not our priority for the moment to be safari compatible.
I close this issue because we moved the repo here https://github.com/SonyCSL-Paris/LeadsheetJS
We will post on https://github.com/SonyCSL-Paris/LeadsheetJS/issues/9 if there is any update on this.
